src/support.js

```diff
@@ -577,16 +577,16 @@ function getFuncWrapper(func, sig) {
   if (!sigCache[func]) {
     // optimize away arguments usage in common cases
     if (sig.length === 1) {
-      sigCache[func] = function dynCall_wrapper() {
```

Comment:
these names have no semantic effect, but they do show up in stack traces while debugging, so maybe worth keeping them?